charts/tfm-almacar/charts/restapi/templates/configmap_properties.yaml

```diff
@@ -0,0 +1,18 @@
+kind: ConfigMap
```

Comment:
Pregunta por ignorancia:
esto se convierte en el application.yaml file que era un fichero físico.
Como se le dice a la magia que este es su nuevo fichero?
Con el fichero bootstrap.properties ?

Reply:
From documentation:

"The default behavior is to create a ConfigMapPropertySource based on a Kubernetes ConfigMap which has metadata.name of either the name of your Spring application (as defined by its spring.application.name property) or a custom name defined within the bootstrap.properties file under the following key spring.cloud.kubernetes.config.name."

Link: https://cloud.spring.io/spring-cloud-static/spring-cloud-kubernetes/2.1.0.RC1/multi/multi__kubernetes_propertysource_implementations.html#_configmap_propertysource

Reply:
great, thx